Stargaze studio (#20)
public-awesome/docs: guides/stargaze-studio/interact-with-a-collection/execute-messages.md

A set of actions can be executed by the admin of the collection. Executable actions are as follows:

* **Mint**

Mint a token from the collection to the caller's address.

* **Purge**

> Purge the data stored inside the collection contract. This action is irreversible and requires the collection to be sold out.

Reviewer: ```suggestion
Purge the minter address data stored inside the minter contract. This action is irreversible and requires the collection to be sold out.
```

* **Batch Transfer Tokens**

Takes an address and multiple token IDs (separated by commas), then transfers all of the tokens to the address. The format for listing token IDs is the same as **Batch Mint For** operation.

* **Burn Token**

Burns the token with the given ID.

* **Batch Burn Tokens**

Takes multiple token IDs and burns them at once. The format for listing token IDs is the same as **Batch Mint For** operation.

{% hint style="danger" %}
Burn actions are irreversible. The burned tokens will never be accessible again.
{% endhint %}

* **Shuffle the token IDs**

Shuffles all of the token IDs inside the collection. Collections are created in the same order as they are sent by the user, by default. Executing this message will shuffle all the IDs to make the orders random.

{% hint style="info" %}
The shuffling price is 500 $STARS.
{% endhint %}

> * **Airdrop Tokens**

Reviewer: Airdrop Tokens now accepts a .csv file, which includes the header "address,amount", to facilitate airdropping multiple tokens to individual addresses.

* **Batch Transfer Tokens**

Takes an address and multiple token IDs (separated by commas), then transfers all of the tokens to the address. The format for listing token IDs is the same as **Batch Mint For** operation.

* **Burn Token**

Burns the token with the given ID.

* **Batch Burn Tokens**

Takes multiple token IDs and burns them at once. The format for listing token IDs is the same as **Batch Mint For** operation.

{% hint style="danger" %}
Burn actions are irreversible. The burned tokens will never be accessible again.
{% endhint %}

* **Shuffle the token IDs**

Shuffles all of the token IDs inside the collection. Collections are created in the same order as they are sent by the user, by default. Executing this message will shuffle all the IDs to make the orders random.

{% hint style="info" %}
The shuffling price is 500 $STARS.
{% endhint %}

* **Airdrop Tokens**

> Takes a CSV file that contains Stargaze addresses and airdrops a token for each address.

Reviewer: ```suggestion
Takes a CSV file that contains Stargaze addresses and airdrops the specified number of tokens for each address.
```

The expected file is a simple .csv file where each line consists of one Stargaze address then a comma followed by the number of tokens to airdrop. The file should look like the following:

```

> stars1arkrg6nydm9k6d2hwzn9rdqtjtfes58hfqhena, 3

Reviewer: ```suggestion
address,amount
stars1arkrg6nydm9k6d2hwzn9rdqtjtfes58hfqhena,3
```

> stars1d90w4s4pcup6qceyrvckj35zwwy2j4u2rqa43t, 1

Reviewer: ```suggestion
stars1d90w4s4pcup6qceyrvckj35zwwy2j4u2rqa43t,1
```
